src/ensembl/doc/architecture/decisions/0004-unit-tests.md

```diff
@@ -1,6 +1,6 @@
 # 4. Unit Tests
 
-Date: 2018-08-30
+Date: 2018-08-30; Last update date: 2019-04-09
 
 ## Status
 
```

Comment:
We should link to 014 so that it's clear we're not using Enzyme

Reply:
@ens-st3, added the link in https://github.com/Ensembl/ensembl-client/pull/407/commits/81d63e87641be9fc44578ed58100a1f85dbf05bd